nowcasting_dataset/manager/README.md

```diff
@@ -0,0 +1,53 @@
+# nowcasting_dataset
+
+This main dir contains the following dirs and files
+
+## Dirs
+
+### base.py
+
+Base class used for `Manager` and `ManagerLive`
+
+This contains
+1. `load_yaml_configuration` - loads a configuration file
+2. `initialise_data_sources` - Initialise all the data sources. Note that this can be the full set of data sources, or a subset can be passed in
+3. `filename_of_locations_csv_file` - get the file name of the locations csv.
+
+### manager.py
+
+Class used to make batches from the different data sources
+
+![alt text](manager.png "Manager")
+([here](https://docs.google.com/presentation/d/10P0-EikNrJrGWzCrGEcXoktLTAwp8n7I-fwTT4IfNso/edit#slide=id.p) for raw image)
+
+In general there are a few steps how to use this function
+1. `load_yaml_configuration` - loads a configuration file. This is from `base.py`.
+2. `initialise_data_sources` - Initialise all the data sources. Note that this can be the full set of data sources, or a subset can be passed in. This is from `base.py`.
```

Comment:
Would it be possible to change this to American English? Most of the rest of the code should be with American spelling
```suggestion
2. `initialize_data_sources` - Initialise all the data sources. Note that this can be the full set of data sources, or a subset can be passed in. This is from `base.py`.
```